docs/workflow/building/coreclr/nativeaot.md

```diff
@@ -55,7 +55,7 @@ For the compilers:
 - Configure the LLVM source to use the same runtime as the Jit: `cmake -G "Visual Studio 17 2022" -DCMAKE_BUILD_TYPE=Debug -D LLVM_USE_CRT_DEBUG=MTd path-to-the-build-directory` or if building for the Release configuration `cmake -G "Visual Studio 17 2022" -DCMAKE_BUILD_TYPE=Release -D LLVM_USE_CRT_RELEASE=MT path-to-the-build-directory`
 - Build LLVM either from the command line (`cmake --build . --target LLVMCore LLVMBitWriter`) or from VS 2022. Currently the Jit depends only on the output of LLVMCore and LLVMBitWriter projects.  For the Release configuration, `cmake --build . --config Release  --target LLVMCore LLVMBitWriter`
 - Set the enviroment variable `LLVM_CMAKE_CONFIG` to locate the LLVM config: `set LLVM_CMAKE_CONFIG=path-to-the-build-directory/lib/cmake/llvm`. This location should contain the file `LLVMConfig.cmake`.
-- Build the Jits and the ILC: `build clr.jit+clr.wasmjit+clr.tools -c [Debug|Release]`. Note that `clr.jit` only needs to be built once.  Add the `clr.nativeaotlibs+libs` subsets if you want the packages for publishing, e.g. `build clr.jit+clr.wasmjit+clr.tools+clr.nativeaotlibs+libs -c Debug`
+- Build the Jits and the ILC: `build clr.jit+clr.wasmjit+clr.tools+clr.nativeaotlibs -c [Debug|Release]`. Note that `clr.jit` only needs to be built once.  Add the `libs` subsets if you want the packages for publishing, e.g. `build clr.jit+clr.wasmjit+clr.tools+clr.nativeaotlibs+libs -c Debug`
```

Comment:
```suggestion
- Build the Jits and the ILC: `build clr.jit+clr.wasmjit+clr.nativeaotlibs+clr.tools -c [Debug|Release]`. Note that `clr.jit` only needs to be built once.  Add the `libs` subsets if you want the packages for publishing, e.g. `build clr.jit+clr.wasmjit+clr.tools+clr.nativeaotlibs+libs -c Debug`
```
Presumably the order is important.

Reply:
The order of subsets on the build command line is not important. The subsets are mapped to what to build in https://github.com/dotnet/runtime/blob/1529058b7e6b7a13253278f45e28f73c04149896/eng/Subsets.props#L336-L370 . This mapping will create the right order.

Reply:
I see. I suppose there is only aesthetic benefit to having dependencies come before dependents then.